website/static/js/koHelpers.js

```diff
@@ -69,6 +69,27 @@ var sanitizedObservable = function(value) {
     });
 };
 
+//Maps an object two deep into a given object as ko observables.
+var mapJStoKO = function(data, parent, array, deep) {
```

Comment:
Rather than mutating `parent`, it would be simpler if this function just took an object as input and another object (with observables) as output. We could then do:

```js
$.extend(self, mapJStoKO(data));
```